include/ck_tile/ops/gemm/kernel/gemm_kernel.hpp

```diff
@@ -365,14 +437,15 @@ struct GemmKernel
                 return false;
             }
         }
-        return true;
+        return DTesnorIsValid && true;
     }
 
     template <memory_operation_enum DstInMemOp = memory_operation_enum::set>
     CK_TILE_DEVICE static auto MakeGemmTensorViews(const ADataType* a_ptr,
                                                    const BDataType* b_ptr,
-                                                   CDataType* c_ptr,
-                                                   const GemmKernelArgs& kargs,
+                                                   const DsGridPointer ds_ptr,
+                                                   EDataType* e_ptr,
+                                                   const GemmKernelArgs<DsGridPointer>& kargs,
```

Comment:
Why we would actually need here anything apart from `GemmKernelArgs` ? It holds all pointers and strides and ds pointers...